src/main/java/net/atos/zac/zoeken/IndexeerService.java

```diff
@@ -161,29 +168,42 @@ public int indexeer(final int batchGrootte, final ZoekObjectType type) {
         return countZoekIndexEntities(type);
     }
 
-    private void readAllZakenAndMarkToAddOrUpdateInSolrIndex() {
+    private void markAllZakenForReindexing() {
         final ZaakListParameters listParameters = new ZaakListParameters();
         listParameters.setOrdering("-identificatie");
         listParameters.setPage(FIRST_PAGE_NUMBER_ZGW_APIS);
         boolean hasNext = true;
         while (hasNext) {
             final Results<Zaak> results = zrcClientService.listZaken(listParameters);
             results.getResults()
-                    .forEach(zaak -> createZoekIndexEntityAddOrUpdate(zaak.getUuid().toString(), ZoekObjectType.ZAAK));
+                    .forEach(zaak -> markItemForAddOrUpdateInSolrIndex(zaak.getUuid().toString(), ZoekObjectType.ZAAK));
             hasNext = results.getNext() != null;
             listParameters.setPage(listParameters.getPage() + 1);
         }
     }
 
-    private void readAllTakenAndMarkToAddOrUpdateInSolrIndex() {
+    private void markAllInformatieobjectenForReindexing() {
+        final EnkelvoudigInformatieobjectListParameters listParameters = new EnkelvoudigInformatieobjectListParameters();
+        listParameters.setPage(2); // niet FIRST_PAGE_NUMBER_ZGW_APIS omdat er op pagina 1 en 2 documenten zonder inhoud zitten en open-zaak een 500 terug
```

Comment:
Heb je hier al een issue voor aangemaakt? Of zet er anders een verwijzing in naar het issue voor het opschonen van Open Zaak en zet in het issue van het Opschonem van Open Zaak dat er een ToDo in de code staat met een verwijzing naar het issue

Want hoe gaan we er voor zorgen dat dit niet vergeten wordt !!!!!

Reply:
Ik heb het issue opgenomen in een todo, en in het issue vermeld dat na het oplossen van het issue dit moet worden aangepast